src/__tests__/Search.test.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Search from "../components/Search";
+
+describe("Search", () => {
+  const validProps = {
+    setResults: jest.fn(),
+  };
+
+  const { asFragment } = render(<Search {...validProps} />);
+
+  test("renders correctly", () => {
```

Comment:
Could another test be added here to check all the parts of the component render as they are supposed to? Could check for the placeholder text in the search form for example